src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -2673,10 +2671,18 @@ def cli(self, vrf_type='all', vrf='all', instance='all', neighbor='', address_fa
                     sub_dict['link_state'] = m.groupdict()['link_state']
                 sub_dict['local_as_as_no'] = int(m.groupdict()['local_as_as_no'])
 
-                # Default the values - overwritten if configured
-                sub_dict['local_as_no_prepend'] = False
-                sub_dict['local_as_replace_as'] = False
-                sub_dict['local_as_dual_as'] = False
+                if m.groupdict()['local_as_no_prepend']:
```

Comment:
thanks for fixing/updating this. Can I ask you to add UT with output from device?

Reply:
Am I mistaken that there's currently no UT or folder based test at all for this command?

Reply:
there is as old style UT.
https://raw.githubusercontent.com/CiscoTestAutomation/genieparser/master/src/genie/libs/parser/iosxr/tests/test_show_bgp.py

But if you can, please add UT for this as folder based test.

Reply:
I've updated the old-style test, but I really don't have the capacity to make a whole new folder based test for this command.

Reply:
Update. I think I've got this done. The folder-based test works on my local system.

<img width="848" alt="Screen Shot 2021-08-03 at 11 03 04 AM" src="https://user-images.githubusercontent.com/6683984/128048432-b06b2d2a-8b5b-4585-b28c-15cfeb002706.png">

Reply:
thanks. yes, only what you developed is fine.